### ISSUE TITLE
Support P9813 chainable LEDs.

### ISSUE THREAD

**Issue by ebaauw** [ebaauw/homebridge-rpi#148]
See Discord.

P9813 looks very similar to the Blinkt!, but uses a slightly different encoding of the LED target state.  See https://github.com/topics/p9813.

**Comment by ebaauw** [comment]
Received the two [Grove chainable LEDs](https://www.seeedstudio.com/Grove-Chainable-RGB-Led-V2-0.html?queryID=0a3c347d25e89f1141955fb2c6d24e4c&objectID=282&indexName=bazaar_retailer_products) I ordered, and hooked them up to my Raspberry Pi.  The card that came with LEDs mention that they could be driven by either 3.3V or 5V, so I just used 3.3V for power.

Indeed, as reported on Discord, Homebridge RPi controls the colour when controlling the LEDs as `blinkt` device, but doesn't provide brightness nor on/off control.
